### PR TITLE
Add Django 5.0 and Python 3.12 to test matrix

### PR DESCRIPTION
**Pull request by mgrdcm**
The fix in https://github.com/jazzband/django-embed-video/pull/201 is required for Python 3.12 so wanted to add testing to cover that, and adding Django 5.0 support at the same time.

Should probably remove support for unsupported Django and Python versions, but that might require a breaking change version bump.  Hoping we can get a release out soon that has the https://github.com/jazzband/django-embed-video/pull/201 change in it, and *then* remove support?

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -2,8 +2,9 @@
 envlist =
     py{37,38,39,310}-dj32
     py{38,39,310}-dj40
-    py{38,39,310,py311}-dj41
-    py{38,39,310,py311}-dj42
+    py{38,39,310,311}-dj41
+    py{38,39,310,311}-dj42
+    py{310,311,312}-dj42
```
Comment (aleksihakli) [inline tox.ini:7]: Should this be `py{310,311,312}-dj50`?

**Review by aleksihakli — CHANGES_REQUESTED** [review]
Looks good, should `tox.ini` be updated to include `dj50` in the matrix as well?

**Comment by aleksihakli** [comment]
Yeah, let's bake a release after this is merged 👍

File: tox.ini (hunk 1 above)
Comment (mgrdcm) [reply tox.ini]: 🤦 Yes, oops, fixed here: https://github.com/jazzband/django-embed-video/pull/204/commits/96bae53fa73970e0228e72561ffd072a91a31963

**Comment by mgrdcm** [comment]
> Yeah, let's bake a release after this is merged 👍

Awesome!  Fixed the one thing you mentioned.  I suppose I could merge it but probably best you do since it's my PR :)

I'm also not familiar with how to get a release of this out, are there instructions?  Or if you're able to just do it, that works too and thank you!

**Pull request merged**